### DISCUSSION TITLE
Edit menu

### DISCUSSION THREAD

**Discussion by sayya112** [moudey/Shell#553] [Q&A]
Hi, Just installed this and trying to figure out how to do thing. No coding experience. I trying to remove certain items from menu after I right click on a file. I would like to remove all the highlighted items from the menu when I right click. Is there GUI version where I can do this? or i can only edit it out in notepad? thanks you for the help

![Screenshot 2024-11-15 155244](https://github.com/user-attachments/assets/52c4028e-52c3-4f79-a872-dbec3bf611ba)

**Comment by ghost** [comment]
Open `shell.nss` with notepad. For each item add this at the end:
```js
remove(find='item_to_remove' type='file')
```
replace `item_to_remove` with the name of the items you want to remove.
